frontend/src/components/header/userMenu/confirmLogOut/props.ts

```diff
@@ -0,0 +1,5 @@
+export interface ConfirmLogOutProps {
+    handleLogOut: () => void;
+    showDialog: boolean;
+    handleClose: () => void;
```

Comment:
I think handleClose and showDialog can be controlled inside of ConfirmLogOut component. Only handleLogOut should be exposed